services/wordpress/wordpress-version-color.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const { promisify } = require('util')
+const semver = require('semver')
+const { regularUpdate } = require('../../core/legacy/regular-update')
+
+// TODO: Incorporate this schema.
+// const schema = Joi.object()
+//   .keys({
+//     offers: Joi.array()
+//       .items(
+//         Joi.object()
+//           .keys({
+//             version: Joi.string()
+//               .regex(/^\d+(\.\d+)?(\.\d+)?$/)
+//               .required(),
+//           })
+//           .required()
+//       )
+//       .required(),
+//   })
+//   .required()
+
+function getOfferedVersions() {
+  return promisify(regularUpdate)({
+    url: 'https://api.wordpress.org/core/version-check/1.7/',
+    intervalMillis: 24 * 3600 * 1000,
+    json: true,
+    scraper: json => json.offers.map(v => v.version),
+  })
+}
+
+async function versionColorForWordpressVersion(version) {
+  const offeredVersions = await getOfferedVersions()
+
+  // What is this?
```

Comment:
🤣

Reply:
I think this code has now been moved twice, but probably it should be completely rewritten. This should also have some of its own tests. There is already a lot changing in this PR. I'm not sure it needs to be dealt with now.